firmware/integration/rusefi_config.txt

```diff
@@ -1102,7 +1102,12 @@ int16_t tps2Max;Full throttle#2. tpsMax value as 10 bit ADC value. Not Voltage!\
 	custom can_baudrate_e 1 bits, U08, @OFFSET@, [0:1], @@can_baudrate_e_enum@@
 	can_baudrate_e canBaudRate; set can_baudrate
 
-	uint32_t[5] unused_former_warmup_target_afr;;"units", 1, 0, -20, 100, 0
+	uint32_t verboseCan2BaseAddress;;"", 1, 0, 0, 536870911, 0
```

Comment:
do not get it. you remove 4*5 bytes and insert x+4*31 bytes, this would increase the total size of configuration?